cfpb/grad-path
# Use consumerfinance.gov public APIs

Now that the API endpoints on consumerfinance.gov have been opened up to the public, we can use those instead of a locally-running instance!

## Changes

- API endpoint base changed from localhost:8000 to consumerfinance.gov

## Testing

1. Start the app per instructions in the README
2. Use the tool, inspecting the API requests in the network panel
3. See that the requests go to consumerfinance.gov and succeed

## Notes

- Also updates the version in `.nvmrc` which was previously overlooked
